### DISCUSSION TITLE
【翻訳案】The Frosty Planet Pack 初出の単語

### DISCUSSION THREAD

**Discussion by nsk4762jp** [nsk4762jp/OxygenNotIncluded-Japanese#375] [Ideas]
DLC初出の単語について、この場でいろいろ検討できたらなと思います。
まだ一覧化できてませんが、随時リストアップしていく予定です。

### 検討中

原語 | 翻訳案 | 備考
----- | ----- | -----

### 確定

原語 | 翻訳案 | 備考
----- | ----- | -----
Alveo Vera | エアロベラ |  アロエベラ(Aloe Vera)の捩り
Bammoth | バンモス | 毛むくじゃらの大型動物
Bammoth Patty | バンモスの糞 | タグは POOP なので牛糞
Bonbon Tree | ボンボンツリー | 
Cinnabar Ore | 辰砂鉱石 | 
Cool Pool | 寒冷湖 | 
Cooling Water | 水を冷却中 | 
Deep Fried Steak | 揚げステーキ | 
Deep Fryer | フライヤー | 
Fish Taco | フィッシュタコス | 
Flox | フロックス | 鹿もしくは角の生えた狐
Geo Vents | 地熱噴出孔 | 
Geothermal Heat Pump | 地熱ヒートポンプ | 
Ice Cave | 氷洞 | 初期バイオームの名前(CLUSTER_NAMES カテゴリに影響)
Ice Liquefier | 融氷器 | ヤカン
Idylla Flower | ボッカ | Idyll(牧歌) + Flower(花) 
Lumen Quartz | ルーメン水晶 | 
Material Content Monitor | 素材コンテンツモニター | 
Mercury Gas | 水銀(気体) | 
Mercury ceiling lights | 水銀天井灯 | 
Nectar | ネクター | 
Nosh Noms | ノッシュ豆のカリカリ揚げ | 
Oxylite Sconce | オキシライト燭台 | 
Pemmican | ペミカン | 
Pikeapple Bush | パイクップルの低木 | パイナップル(Pineapple)の捩り
Pikeapple Skewer | パイクップルの串焼き |
Plume Squash  | ハネナンキン | 
Research Decrypter | 研究用解読器 | 
Research Portal | 研究ポータル | 
Shellfish Tempura | 剥き身の天ぷら | 
Sherberry | シャーベリー | 
Snow Tile | 雪製タイル | 
Spigot Seal | パッキンアシカ | Spigot(`蛇口`)、Seal(`アシカ`/`アザラシ`)
Squash Fries | ナンキンフライ | 
Tallow | 獣脂 | 
Warm Coat | あったかコート | 
Wood Heater | 薪ヒーター | 
Wood Pile | 木材の山 | 
Wood Tile | 木製タイル |

**Comment by KoheiKanagu** [comment]
`STRINGS.BUILDING.STATUSITEMS.COOLINGWATER.NAME`
Cooling Water
案: 冷却水

**Reply by nsk4762jp** [reply]
DESC のほうには `この設備は水を凝固点まで冷却しています` とあるので、「冷却するための水」ではなく「冷却中の水」を指すものと思われます。

**Comment by KoheiKanagu** [comment]
Ice Liquefier
案: 氷液化装置

内部的には `ICEKETTLE` のようです
例のヤカンですね
<img width="111" alt="スクリーンショット 2024-07-19 13 53 34" src="https://github.com/user-attachments/assets/4f0e91d6-b342-4cb9-8203-2b74daa36d4f">

**Reply by denev-rove** [reply]
`融氷器`というのはどうでしょうか。
固体を液体に変える場合、日本語だと「融かす」と表現したほうが直感的かなと思います。（融雪剤とか）
個人的な印象かもしれませんが、「液化」は気体を液体に変える感じがします。

**Comment by KoheiKanagu** [comment]
Wood heaters

いわゆる 薪ストーブ の類だと思いますが、 "薪ストーブ" は既にあるので別の言い方が必要

<img width="123" alt="スクリーンショット 2024-07-19 14 35 38" src="https://github.com/user-attachments/assets/7f4fdb44-79f3-49d0-8eb6-3333336644b5">

**Reply by nsk4762jp** [reply]
Wood Heater も Wood Burner も単語上は同じ意味(`薪ストーブ`)なので、名前を付けるとしたら効果別にするしかなさそうですね。
ストーブより火力が高いのであれば純粋に `薪ヒーター` かな。

**Reply by KoheiKanagu** [reply]
熱量としてはWood Heaterの方が強いですね （薪ストーブに比べて熱量あたりの木材のコスパすごい...）

<img width="942" alt="スクリーンショット 2024-07-19 14 59 15" src="https://github.com/user-attachments/assets/9fd7af50-7c5c-4732-bd3f-fe23ac33eac3">
<img width="949" alt="スクリーンショット 2024-07-19 14 59 28" src="https://github.com/user-attachments/assets/cffc3cb9-403c-48f8-b6d1-518f89e82fa9">

**Reply by nsk4762jp** [reply]
こりゃ凄い。薪ストーブの完全上位互換みたいな性能してますねー 😄 
薪暖房か薪ヒーターにしますか。

**Comment by KoheiKanagu** [comment]
Research Portal
案: 研究用ポータル

research decrypter
案: 研究用解読器

<img width="1575" alt="スクリーンショット 2024-07-19 14 51 37" src="https://github.com/user-attachments/assets/c489191e-e7e8-4426-b7c2-f60d3947c337">

**Comment by KoheiKanagu** [comment]
Geo Vents
案: 地熱噴出孔

**Comment by KoheiKanagu** [comment]
Geothermal Heat Pump
案: 地熱ヒートポンプ

**Comment by KoheiKanagu** [comment]
Material Content Monitor
案: 素材コンテンツモニター

（Geo Vents関連のようですが詳細不明）

**Comment by KoheiKanagu** [comment]
Mercury ceiling lights
案: 水銀シーリングライト

https://oxygennotincluded.wiki.gg/wiki/Mercury_Ceiling_Light
![image](https://github.com/user-attachments/assets/9c9b3acb-7123-47bd-adeb-a60bec3a714a)

**Reply by denev-rove** [reply]
既存の`Ceiling Light` は `天井灯` と訳されてますね。
ほかとの混同がなければ、単に `水銀灯` でも良いように思います。

**Reply by KoheiKanagu** [reply]
確かに `天井灯` ありますね。失礼しました
`水銀灯` はどちらかというと Lamp のイメージがあります

既存の言い方に倣うなら `水銀天井灯` ですが語感が悪いような気も...

**Reply by denev-rove** [reply]
私も語感の悪さを嫌って`水銀灯`を提案してみましたが、`水銀天井灯`でも全く問題ない（すぐに慣れる）と思います。

**Comment by KoheiKanagu** [comment]
Snow tiles
案: 雪タイル

https://oxygennotincluded.wiki.gg/wiki/Snow_Tile
![image](https://github.com/user-attachments/assets/8347d8a8-8287-49df-8afd-28687e8e940f)

**Reply by sugarlessboy** [reply]
細かくて恐縮ですが、`木製タイル` `プラスチック製タイル` に倣って `雪製タイル` がよいと思いました。

**Comment by KoheiKanagu** [comment]
wood tile
案: 木製タイル

https://oxygennotincluded.wiki.gg/wiki/Wood_Tile
![image](https://github.com/user-attachments/assets/c52b008a-f6f0-406e-abcb-641aa818966d)

**Comment by KoheiKanagu** [comment]
Tallow
案: 獣脂

https://oxygennotincluded.wiki.gg/wiki/Tallow
![image](https://github.com/user-attachments/assets/2378c9b7-eb77-4787-b658-40e20538befd)

**Reply by nsk4762jp** [reply]
参考: [ヘット(牛脂)](https://ja.m.wikipedia.org/wiki/%E3%83%98%E3%83%83%E3%83%88) も Tallow というそうです。ラードは豚脂です。

**Comment by KoheiKanagu** [comment]
Deep Fryer
案: フライヤー

https://oxygennotincluded.wiki.gg/wiki/Deep_Fryer
![image](https://github.com/user-attachments/assets/f9865c7b-7327-495d-a9fb-1d179eacb5bd)

**Reply by denev-rove** [reply]
`揚げ物調理器`などはどうでしょうか。
今の日本語だとカタカナ語の`フライヤー`は`Fryer`の意味でも`Flyer`（チラシ）の意味でも使われているので、少し紛らわしいかなと。

**Reply by nsk4762jp** [reply]
Steam Workshop のコメント欄から

> Deep Fryerも「フライヤー」で十分わかります。揚げ物調理器でもいいですが、チラシの意味でフライヤーを使うより一般的に広く浸透しているのは調理器具のほうかと。理由がそれなら漢字にする意義を感じません。

**Comment by KoheiKanagu** [comment]
Wood Pile
案: 木材の山

![image](https://github.com/user-attachments/assets/db7b398c-3b10-4da9-a0a4-e95649ca9847)

**Reply by nsk4762jp** [reply]
これまでは `Lumber` を `木材` と訳していましたが、今回のアップデートで `Lumber` という語自体が `Wood` に置き換わっているっぽいですね。

**Reply by KoheiKanagu** [reply]
そのようです
全部 `Wood` に寄せられてますね

**Reply by nsk4762jp** [reply]
あまり議論の余地がなさそうなものは(こちらの判断で)確定とさせていただきます。

**Comment by KoheiKanagu** [comment]
Oxylite Sconce
案: オキシライト燭台

![image](https://github.com/user-attachments/assets/e124a3b7-8d39-4fd7-a4c5-fbd09961d41d)

**Comment by denev-rove** [comment]
Alveo Vera
案: 肺胞アロエ

https://oxygennotincluded.wiki.gg/wiki/Alveo_Vera
![Alveo_Vera](https://github.com/user-attachments/assets/02223274-42be-44f2-9dc6-325d4683ca22)

**Reply by denev-rove** [reply]
アロエの一種である[アロエベラ](https://ja.wikipedia.org/wiki/%E3%82%A2%E3%83%AD%E3%82%A8%E3%83%99%E3%83%A9)のもじりでしょうか。
Alveo は 肺胞という意味のようです。データベースの説明でも、幹(？)にある小孔から二酸化炭素を吸って、オキシライト（酸素鉱石）を生成するとあります。

**Reply by KoheiKanagu** [reply]
（オキシライトこねくり回しながら大きくするアニメーションでしたけど、そう言うことですか）
~穴あきのアロエか気持ち悪い... でも装飾値は+15~

**Reply by nsk4762jp** [reply]
Steam Workshop のコメント欄から
> - Alveo Veraも忠実に肺胞とするより捩りをとってエアロベラとかにしてもいいんじゃないでしょうか
> - Alveo Veraも「肺胞」と言われると酸素を吸って二酸化炭素を出す、という印象を受けるので、それなら他の方が提案している「エアロ」のほうが納得感があって好きです。

**Reply by denev-rove** [reply]
なるほど、アロエ → エアロは上手い捩りですね。
私は名前だけでそれが何なのかすぐに分かるのが好きなので、アロエという文言は残したいですが、最終判断はお任せします。
エアロを採用するなら、単語の区切りとして空白があったほうが分かりやすいと思います。`エアロ ベラ`

**Reply by denev-rove** [reply]
エアロベラは空白なしで決まりでしょうか。「エアロベラ」だと、そういう固有名詞だと思われる可能性もあると思います。アロエベラが誰でも知ってるような品種だったら良かったんですが。

**Comment by KoheiKanagu** [comment]
Cinnabar Ore
案: 辰砂鉱石

`しんしゃ` と読むようです
https://ja.wikipedia.org/wiki/%E8%BE%B0%E7%A0%82

https://oxygennotincluded.wiki.gg/wiki/Cinnabar_Ore
![image](https://github.com/user-attachments/assets/da133e0d-536d-4a33-adfa-7e7eef710f79)

**Reply by souki-tn** [reply]
私の翻訳した範囲で一部「朱色」と訳しているところがあります。掛け軸とかの落款に辰砂が使われるので、色で表現しましたが、辰砂に戻した方が良いかもしれません。

**Comment by KoheiKanagu** [comment]
Mercury Gas
案: 水銀(気体)

**Comment by KoheiKanagu** [comment]
Nectar
案: ネクター

https://oxygennotincluded.wiki.gg/wiki/Nectar

Bonbon Tree から取れるようですが樹液ならSapだと思うので、Bonbon Treeの花の蜜のようです
`ネクター` だとフルーツのジュースっぽい気はします

**Reply by denev-rove** [reply]
ネクターは英語読みで、日本のゲーム等では本来の`ネクタル`と読ませることが多いように思います。
たしかに不二家の印象が強すぎることも影響してる気がします。

**Reply by nsk4762jp** [reply]
そのまま `ネクター` とするか、日本語訳した `甘露` あたりになるかなと思います。

**Reply by souki-tn** [reply]
甘露だと、上等なお茶になっちゃうので、カタカナの方が良いと思います。

**Comment by KoheiKanagu** [comment]
Warm Coat
案: あったかコート

<img width="257" alt="スクリーンショット 2024-07-19 23 20 41" src="https://github.com/user-attachments/assets/9691d3b8-c2f9-4ac3-b5b6-744dc5e60b83">

`Warm Sweater: あったかセーター` は無くなってコートに置き換えられたようです

**Comment by sugarlessboy** [comment]
Pikeapple Bush
案：パイカップルの低木
![Pikeapple_Bush](https://github.com/user-attachments/assets/b7fc7c10-e96e-4cc9-bde9-6ec6b17f3992)

`Bush`なので、「茂み」「藪」も考えつつ、グラフィックを観る限り「低木」が一番近そうだと感じました。
（パイカップルの語感、素敵です😊）

**Reply by nsk4762jp** [reply]
Steam Workshop のコメント欄から
> Pikeapple、発音よりも表記でパイクアップルと書いたほうがとげとげしているという感じは出る気がします

**Reply by souki-tn** [reply]
もとの綴りをややスポイルしますが、日本語でわかりやすいのであれば「パイクップル」が良いのではないでしょうか。「パイク」をそのまま音として残しつつ、パイナップルとの関連性強めで。
パイクアップルにしてしまうと「パイク＋アップル」と日本語では認識されるので、この場合はよくないかと。「パイカ」にしてしまうと「パイク」であると認識するのは日本語では難しい。

**Comment by sugarlessboy** [comment]
Idylla Flower
案：ボッカ
![Idylla_Flower](https://github.com/user-attachments/assets/b87c7e44-11c2-495e-8c1b-b7b8be232ab2)

`Idylla`は`Idyll`が語源と推察し、「牧歌」+「花」の造語としました。
観葉植物の通例に沿えばカタカナ表記の「アイディラフラワー」になると思いますが、文字数がちょっと多いかなと……。
攻めた翻訳案なので、違和感あればスルーいただいて結構です🙇🏻‍♂️

**Reply by denev-rove** [reply]
攻めた訳、いいですね！
カタカナで`ボッカ`だと単なる固有名詞にも見えるので、造語するなら漢字で`牧花`（`牧花の種`）としたほうが私は好きです。（今のところ私はこれを推します。）

意味を諦めて音写するなら `Balm Lily Flower`（`バームリリーの花`）に合わせて `アイディラの花`（`アイディラの種`）でどうでしょう。文字数も少し短いし。

**Reply by nsk4762jp** [reply]
Steam Workshop のコメント欄から

> Idylla Flowerをボッカ案で行くならカタカナ表記を推します。他の観葉植物が軒並みカタカナ表記なので、わざわざ漢字にするくらいならアイディラフラワーでいいかと

**Reply by denev-rove** [reply]
では私も原案どおりの`ボッカ`を推すことにします。

**Reply by souki-tn** [reply]
意味から掛けるのなら「牧歌」よりも「のどか」の方が語としては一般的なので、「ボッカ」よりは「のど花」の方がいいと思います。
ですが、翻訳は捻りすぎない方が良いことはわかっているので、「アイディラの花」を推します。

**Comment by denev-rove** [comment]
Plume Squash
案: プルーム人参
![plume_squash](https://github.com/user-attachments/assets/b9632927-9ebc-4008-bf5c-cde0624fb2b6) ![plume_squash_plant](https://github.com/user-attachments/assets/a8629bba-0365-405e-b4bc-62b920c82fc7) ![plume_squash_plant2](https://github.com/user-attachments/assets/224e315c-795d-49f9-a594-f87030af947a)

Plume: 飾り羽（孔雀の羽とか、ロビンフッドの帽子に付いてる羽根とかのイメージ）
Squash: （広い意味での）カボチャ
参考: https://www.nikkei.com/article/DGXNASFK2800B_Q3A930C1000000/
参考: https://x.com/June_lotus/status/1585294678227705856

Squash と言いつつ、大根みたいに地下に埋まって育つし、説明文ではtuber（地下茎→ジャガイモなど）になってるし、謎まみれの植物です。Plume の由来は、地上に出てる葉の部分が羽根みたいに見えるからでしょうか。

農作物であることが一目で分かるように、Squash の部分だけ訳すのはどうでしょうか。
（内部のタグでは`CARROT`となっているという指摘を受けて、`人参`という意訳を提案します。）

**Reply by nsk4762jp** [reply]
これタグ上は `CARROT` となっていて人参なんですよね。グラフィックも多分それに倣っていると思います。

**Reply by denev-rove** [reply]
なるほど、大根じゃなくて人参でしたか。確かにそのように見えます。
それでは提案も人参に変えたいと思います。

最終形からは、なんとなく開発者の苦心の跡が見られるような…。

**Reply by souki-tn** [reply]
plume thistlesと称される植物はあるようで、その場合「羽アザミ」と翻訳されるようです。
ただ、種名ではなく漠然としたグループの総称であるようで、無視してアザミと訳すことも多いようで。

**Reply by nsk4762jp** [reply]
![image](https://github.com/user-attachments/assets/fbe28409-33b5-41ce-bd1b-3bf0d49d52ae)

これを揚げたものは Squash Fries ですが、グラフィックがもろにフライドポテト(French Fries)なので人参と訳していいものか自信がなくなってきました。

**Reply by denev-rove** [reply]
ジャガイモの代わりに人参を使った [Carrot French Fries](https://www.google.com/search?q=Carrot+French+Fries&udm=2) (あるいは単に[Carrot Fries](https://www.google.com/search?q=Carrot+Fries&udm=2)) というのもあるので、さほどおかしくはないと思います。まぁ元から名前と実態が一致してない作物なので、突き詰めて考えると、どこかで違和感は出てしまうのかなと。（むしろそうした違和感こそが、この作物のコンセプトなのかも）

得体の知れなさを逆手に取って、そのまま`フライドポテト`としてしまう（芋の作物ということにする）のも確かに有りだと思います。個人的には、野菜（収穫作物）であることさえ示せれば十分だと思っています。

**Reply by souki-tn** [reply]
難しいところですが、Squashは擬音語で押しつぶしてペチャンコにする音に使われます。スマホなどで出ている Plants vs Zombies でSquashはゾンビを叩き潰すカボチャになっています。これも洒落になっているネタのようです。
Squashはその音から、スポーツのスカッシュもこの綴りです。これも擬音語由来です。
ズッキーニなどもSquashの一種と扱われるようで、瓜科の植物のうち広い範囲をSquashと呼ぶようなので、カボチャと翻訳すると色々とイメージが違うことになりそうです。
Squash FriesはFrench Friesに芋ではなく、ズッキーニやカボチャを使うレシピもないことはないようで、そこら辺からイメージになっていそう。Squash Friesで検索すると結構出てきます。

ちょっと苦しい翻訳になりますが、土に埋まっている豆といえばピーナッツ、別名「南京豆」、カボジャの別名といえば「南京」。
「ナンキン」ならやや正体不明で土に埋まっていても良いかもしれないということで「ハネナンキン」なんてどうでしょうか。
Squash Friesの方は「フライドナンキン」になります。

**Reply by nsk4762jp** [reply]
ナンキンは割と妙案だと思うのでそれにしましょう。

**Comment by denev-rove** [comment]
Ice Cave
案: 氷洞
![IceCave](https://github.com/user-attachments/assets/68d92487-9824-4f9f-9349-3ca72f46fcb4)

新バイオームです。
Flosty DLC では、ここが開始地点となるようです。

`Flox`、`Alveo Vera`などが生息する極寒の地形です。（-30℃～-20℃くらい）

**Comment by denev-rove** [comment]
Cool Pool
案: 寒冷湖
![CoolPool](https://github.com/user-attachments/assets/8015dd7e-0ea4-4833-a073-0a188c76f2d8)

新バイオームです。
`Plume Squash`、`Bammoth`などが生息する極寒の地形です。（-40℃くらい）

当然ながら`水`は全て`氷`になってます。しかし液体の`エタノール`があり、Pool というのはそれを指しているものと思われます。
「寒冷」では済まないほどの極低温ですが、'Cool' の訳ならこんな感じになるかなと。（韻を踏むのはちょっと無理でした。）

**Comment by denev-rove** [comment]
Spigot Seal
案: スピゴットアザラシ
![SpigotSeal](https://github.com/user-attachments/assets/1a5e69ff-89ce-4ab9-8ebb-1dbec0906d66)

`Spigot Seal` というのは、パイプ等の接続部に使う、中身が漏れないようにするための部品みたいです。（ゴムパッキン的な？）
[Seal](https://en.wikipedia.org/wiki/Seal_(emblem)): 密閉する、封印、印章（貴族が手紙の封蝋に押すハンコ）
~~Don't Starve でも見たことあるダジャレですね。~~

`Spigot Seal` という一つの句でダジャレが成立していて、"Spigot" の単語自体に大した意味はないのかも。
なんとか訳せないか考えてみましたが、思いつきませんでした。なにか良い案がある方がいればぜひ聞いてみたいです。

**Reply by KoheiKanagu** [reply]
トレーラーの39秒あたりや説明を見ると Spigot Seal が "ストロー" を刺して Bonbon Tree から Nectar を飲むような生態なので、 木の栓、蛇口のようなニュアンスで Spigot なのかもしれません
https://youtu.be/1l9RLiNHy8I?si=ZyTdY3CI1oh3B0M6&t=39

https://github.com/nsk4762jp/OxygenNotIncluded-Japanese/blob/2bf6a89f769c8f9d24609e84bf4f7fedb2eebf06/po/codex.po#L9835-L9852

**Reply by nsk4762jp** [reply]
[【豆知識】知ってる？アザラシとアシカの違い](https://kyoto.uminohi.jp/information/azarashi/)

> 一番わかりやすい違いは、アザラシは歩けないというところ。

ゲーム中の彼らが歩けないってことは無いと思うので、`スピゴットアシカ` または `蛇口アシカ` ですね。

**Reply by denev-rove** [reply]
> トレーラーの39秒あたりや説明を見ると Spigot Seal が "ストロー" を刺して Bonbon Tree から Nectar を飲むような生態なので、 木の栓、蛇口のようなニュアンスで Spigot なのかもしれません https://youtu.be/1l9RLiNHy8I?si=ZyTdY3CI1oh3B0M6&t=39

おー、解説ありがとうございます。説明文にしっかり由来が書いてあったんですね。
中に液体が流れる水道管みたいなものをイメージした名前なんだなってことがよく分かりました。

**Reply by souki-tn** [reply]
"Spigot Seal" → 水道のゴムパッキン であることから "Spigot Seal" + "Seal" という意味だと考えて「パッキンアシカ」とかでどうでしょう。
掛かっている言葉は開いてから翻訳したほうが自然な翻訳になると思います。（高校の古文の授業で習った）

**Reply by KoheiKanagu** [reply]
1タイル分は飛ぶぐらいアクティブに動いているのでアシカっぽいですね。耳は無さそうですが

<img width="277" alt="スクリーンショット 2024-07-21 16 25 50" src="https://github.com/user-attachments/assets/d632e09e-c1c8-42d5-ba90-a2f95b3dc8de">
<img width="148" alt="スクリーンショット 2024-07-21 16 25 59" src="https://github.com/user-attachments/assets/fe78d22d-9123-46bd-b728-2349083950b9">
